src/loaders/express.js

```diff
@@ -91,7 +105,7 @@ export default (app) => {
     });
 
     // Health check endpoint
-    app.get("/", (_, res) => res.status(HTTPStatus.OK).json({ "online": true }));
+    app.get("/", (_, res) => res.status(HTTPStatus.OK).send("OK"));
```

Comment:
Why?

Reply:
I think Render required the health check endpoint to return this, I will try again with the previous version